### DISCUSSION TITLE
How can I create custom grid-template-cols?

### DISCUSSION THREAD

**Discussion by rylanharper** [unocss/unocss#2897] [Q&A]
This should be really simple and it works in windicss and tailwind, but when I add in `grid-cols-[1fr,800px,1fr]` using unocss it returns as invalid. Just checking in to see how the community creates custom grid-template-columns. Thanks!:)

> Edit: I found the answer under the [Brack Syntax Spaces](https://unocss.dev/presets/wind#bracket-syntax-spaces) within the Uno docs! I missed reading over that when I was searching for the answer.

**Comment by haobarry** [comment]
thanks
